docs/application/tizen-studio/setup/prerequisites.md

```diff
@@ -3,8 +3,11 @@
 Check the following prerequisites before attempting to install Tizen Studio.
 
 ## Java Development Kit (JDK) Requirements
+> **Note**
+>
+> There is no JDK prerequisite for 3.7 and higher versions of Tizen Studio
```

Comment:
Suggest rephrasing to: 

There is no JDK prerequisite for Tizen Studio version 3.7 and higher.